nulib/meadow
# Update Authoritex to v1.1.0

# Summary 
Fix issue where exact matches for Getty terms were not at the top of the result set

Before:

![SCR-20240502-leaf](https://github.com/nulib/meadow/assets/196872/856105ad-b246-437c-b350-8055b9933485)

After:

![SCR-20240502-lepg](https://github.com/nulib/meadow/assets/196872/88f8fcd0-18f2-4efb-a2f6-d6324375c6d1)

# Specific Changes in this PR
- Update Authoritex to v1.1.0

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test

Edit a work and try to add the controlled term `photographs` from Getty's AAT authority. It should come up first. Do some searches within AAT and other authorities to make sure search results look (and are ordered) as expected.


# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
